versioned_docs/version-8.5/self-managed/tasklist-deployment/tasklist-authentication.md

```diff
@@ -115,14 +115,6 @@ The user or user group can be assigned the following permission:
 
 For more information, visit the [Identity documentation](/self-managed/concepts/access-control/resource-authorizations.md).
 
-## Use Identity JWT token to access Tasklist API
```

Comment:
We shouldn't remove the authentication part (unless it was moved to another section) as customers still need to understand this to use the API, rather we should change the reference from GraphQL to the new API or Tasklist REST API